pages/onboarding_cluster.ipynb

```diff
@@ -0,0 +1,156 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Cluster onboarding\n",
+    "This guide explains how you can onboard to one of the Operate First clusters. First chapter lists different environments and clusters where you can onboard, second explains how you can onboard as user and third how you can onboard yourself creating PR.\n",
+    ":::{seealso}\n",
+    "For more information about the Operate First see [Getting started with Operate First](references:operate-first-start)\n",
+    ":::\n",
+    "\n",
+    "## User\n",
+    "\n",
+    "List of clusters managed by Operate First are listed [here](references:clusters).\n",
+    "To request access to one of these clusters please file a GitHub issue [here](https://github.com/operate-first/support/issues/new?assignees=first-operator&labels=kind%2Fonboarding%2Carea%2Fcluster&template=onboarding_to_cluster.yaml&title=NEW+PROJECT%3A+%3Cname%3E). \n",
+    "\n",
+    "\n",
+    "## Operation\n",
+    "\n",
+    "This part of tutorial servers as a guide for adding cluster scoped resources of `Namespace` and `Group` kinds. Following steps in this guide should result in a PR against the [operate-first/apps](https://github.com/operate-first/apps) repository.\n",
+    "\n",
+    "### Prerequisites\n",
+    "Please fork/clone the [operate-first/apps](https://github.com/operate-first/apps) repository. During this whole setup, we’ll be working within this repository.\n",
+    "\n",
+    "For successful completion of this guide you need to understand what the aim is. Please have prepared following data:\n",
+    "- Name of the onboarded team.\n",
+    "- Desired namespace name. Please use your team name as a prefix. This will make it easier for you to onboard to ArgoCD later on.\n",
+    "- List of users you’d like to add to your team.\n",
+    "- An optional team GPG key, in case you would like to modify the encrypted list of users of your team later on.\n",
+    "\n",
```

Comment:
These are rather inputs, than prereqs. Let's add it as a guide step.

Reply:
Resolved in latest commit.